lib/OpenQA/Task/Job/Limit.pm

```diff
@@ -50,6 +50,10 @@ sub _limit {
     my $storage = $schema->storage;
     my $dbh     = $storage->dbh;
     my ($min_id, $max_id) = $dbh->selectrow_array('select min(id), max(id) from screenshots');
+    if (!defined $min_id || !defined $max_id) {
```

Comment:
Given that the `id` field is a `serial`, the value would always start at `1`. Meaning that `defined` is not needed.